transports/rabbitmq/index.md

```diff
@@ -14,6 +14,8 @@ tags:
 
 Provides support for sending messages over [RabbitMQ](http://www.rabbitmq.com/) using the [RabbitMQ .NET Client](https://www.nuget.org/packages/RabbitMQ.Client/).
 
+WARNING: The transport is not compatible with versions of the RabbitMQ broker earlier than 3.4.0.
```

Comment:
Yeah, I guess that could work.

So, putting two options forward:

> The transport is not compatible with RabbitMQ broker Versions 3.3.X and below.

or

> The transport is only compatible with RabbitMQ broker Versions 3.4.0 and above.

@SimonCropp @Particular/rabbitmq-transport-maintainers @Particular/docs-maintainers any preference?